libs/designsystem/src/lib/components/tabs/tabs.component.scss

```diff
@@ -43,4 +29,18 @@ ion-tabs {
       }
     }
   }
+
+  // move tabs to top of screen if not touch and large device
+  @include media('>=large') {
```

Comment:
Consider merging/moving/nesting rules that uses same media query:
![Uploading image.png…]()